docs/cli/build/in-obs.md

```diff
@@ -0,0 +1,53 @@
+---
+title: in-obs
+date: 2024-06-27
+sidebar_position: 5
+---
+
+# `packit build in-obs`
+
+Submit a build of the present content in the upstream or local repository to  [OBS](https://build.opensuse.org/).
+
+## Requirements
+
+* SPEC file for the project placed in the repository.
+* Packit config file placed in the repository.
+
+
+
+## Tutorial
+
+1. [Place a config file for packit in the root of your upstream repository.](/docs/configuration/)
+
+2. The command below would create a SRPM from the present content of a repo and perform `osc commit` with it. 
+:::note
+You should specify a project `--project` else a project name in this form `home:<me>:packit` would be used. Don't expect to see that project on your [OBS](https://build.opensuse.org/) account .
```

Comment:
I think this is a bit misleading. What is so bad about the project name `home:$username:packit`?

Reply:
If u don't have the project `home:$username:packit`, the build is still commited but u can't see the package on the website

Reply:
No, that shouldn't happen. You should be able to find it eg here: https://build.opensuse.org/project/show/home:rxbryan:packit or https://build.opensuse.org/project/show/home:dancermak:packit

Reply:
touche!
well that settles it.